Microsoft.Toolkit/Extensions/TypeExtensions.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+#nullable enable
+
+namespace Microsoft.Toolkit.Extensions
+{
+    /// <summary>
+    /// Helpers for working with types.
+    /// </summary>
+    public static class TypeExtensions
```

Comment:
Yes exactly, plus it also includes a couple other cool things:
- Primitive types with a C# alias (eg. `int`, `string`, etc.) will be displayed like that instead of the fully qualified name (eg. `System.Int32`)
- Value tuples will be displayed as `(T1, T2, ...)` (instead of `System.ValueTuple'2[T1, T2]`)
- Nullable value types will be displayed as `T?` (instead of `System.Nullable'1[T]`)

Basically, this method should display types just like users type them in code 😄
(plus the namespace info for non built-in types)